src/SlashCommands.js

```diff
@@ -70,6 +70,19 @@ function success(promise) {
 /* eslint-disable babel/no-invalid-this */
 
 export const CommandMap = {
+    shrug: new Command({
+        name: 'shrug',
+        args: '<message>',
+        description: _td('Prepends ¯\\_(ツ)_/¯ to a plain-text message'),
+        runFn: function(roomId, args) {
+            var message = '¯\\_(ツ)_/¯'
```

Comment:
Please read the style guide. Avoid var and use const/let

Reply:
Thanks, I've taken a look at the style guide and fixed the issues reported by CI.